rdkit/Chem/RegistrationHash.py

```diff
@@ -127,19 +126,24 @@ def GetMolLayers(original_molecule: Chem.rdchem.Mol, data_field_names: Optional[
   mol = _RemoveUnnecessaryHs(original_molecule, preserve_stereogenic_hs=True)
   _StripAtomMapLabels(mol)
 
+  cxflag = Chem.CXSmilesFields.CX_ATOM_LABELS | Chem.CXSmilesFields.CX_ENHANCEDSTEREO
```

Comment:
This should be defined at the module level (i.e. as a "global") so that:
1. we have it defined in one place (it will be used multiple times)
2. client code can modify it if they wish

Reply:
I added a global variable for the default cxflag we want to use, then I added optional parameters to the two methods that use them (GetMolLayers and GetTrautomerHash).